common/lib/xmodule/xmodule/modulestore/xml.py

```diff
@@ -468,12 +486,30 @@ def load_course(self, course_dir, course_ids, tracker):
 
                 # VS[compat]: remove once courses use the policy dirs.
                 if policy == {}:
+
+                    dog_stats_api.increment(
+                        'vscompat.deprecation',
+                        tags=(
+                            "course:{}".format(course),
+                            "location:xml_load_course_policy_dir"
+                        )
+                    )
+
                     old_policy_path = self.data_dir / course_dir / 'policies' / '{0}.json'.format(url_name)
                     policy = self.load_policy(old_policy_path, tracker)
             else:
                 policy = {}
                 # VS[compat] : 'name' is deprecated, but support it for now...
                 if course_data.get('name'):
+
+                    dog_stats_api.increment(
+                        'vscompat.deprecation',
+                        tags=(
+                            "action:{}".format(course_data.get('name')),
```

Comment:
Might be good to include the `org` and `course` in here as well, in case `name` isn't uniquely descriptive.